src/fmu/dataio/dataio.py

```diff
@@ -498,6 +509,27 @@ def __post_init__(self) -> None:
         logger.info("FMU context is %s", self.fmu_context)
         logger.info("Ran __post_init__")
 
+    def _validate_access_ssdl_arguments(self) -> None:
```

Comment:
While `access_ssdl` is still allowed and supported, checking of both old and new arguments are given. Don't want that.